Sources/ParseSwift/API/URLSession+extensions.swift

```diff
@@ -39,7 +39,15 @@ extension URLSession {
             do {
                 return try .success(mapper(responseData))
             } catch {
-                if let error = try? ParseCoding.jsonDecoder().decode(ParseError.self, from: responseData) {
+                if var error = try? ParseCoding.jsonDecoder().decode(ParseError.self, from: responseData) {
+                    do {
+                        let json = try JSONSerialization.jsonObject(with: responseData, options: []) as? [String : Any]
+                        if let code = json?["code"] as? Int {
+                            error.intCode = code
+                        }
+                    } catch {
+                        /* */
```

Comment:
Was something suppose to go here?

Reply:
I'm not sure. I'm not very experienced with swift and not sure how the linting works. In JS, the Lint gets mad when you have empty `catch` blocks.

Reply:
Did you install swiftlint? https://github.com/parse-community/Parse-Swift/blob/main/CONTRIBUTING.md

Is it giving you any warnings? There should only be 1 warning in the SDK that shows up in < Xcode 12.5 and is marked with a comment by me. All others should be fixed before merging

Reply:
This block should be written different, the empty catch won’t work

Reply:
I haven't got swiftlint working. I'll work on it.

The block was to get "code" out of the response payload. I couldn't work out how else to do it (I don't actually write in Swift, this work is for a colleague.